app/models/payment.rb

```diff
@@ -40,22 +40,27 @@ def self.build(bt_customer, bt_payment_method, member_id, existing_customer)
 
     def initialize(bt_customer, bt_payment_method, member_id, existing_customer)
       @bt_customer = bt_customer
-      @bt_payment_method = bt_payment_method
       @existing_customer = existing_customer
       @member_id = member_id
+      @bt_payment_method = bt_payment_method
     end
 
     def build
       if @existing_customer.present?
         @existing_customer.update(customer_attrs)
       else
-        Payment::BraintreeCustomer.create(customer_attrs)
+        @existing_customer = Payment::BraintreeCustomer.create(customer_attrs)
```

Comment:
Yeah good point. I'll change that. I just kinda kept the variable names from before.